spec/spec_helper.rb

```diff
@@ -55,13 +71,6 @@
   end
 end
 
-module MultiTenantTest
-  class Application < Rails::Application; end
-end
-
-MultiTenantTest::Application.config.secret_token = 'x' * 40
-MultiTenantTest::Application.config.secret_key_base = 'y' * 40
```

Comment:
This 2 lines are not used. So I removed.

Reply:
Thanks @nipe0324 for your contribution :)